### PR TITLE
chore: add MixPanel info to readme

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Update the README to include information on the developer workflow for MixPanel.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Almost literally 0.

## Testing

N/A

### Engineering

N/A

### Operations

N/A

## Screenshots (if applicable)

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,3 +140,11 @@ yarn build:web
 The command `yarn release` helps to automate the release process.
 
 Run the command and follow the prompts.
+
+## MixPanel
+
+MixPanel is used for non-private deployments of the interaface.
```
Comment (gomesalexandre) [inline README.md:146]:
```suggestion
MixPanel is used for non-private deployments of the [userinyerface](https://userinyerface.com).
```

Comment (gomesalexandre) [reply README.md:146]: or that one
```suggestion
MixPanel is used for non-private deployments of the interface.
```

**Review by gomesalexandre — APPROVED** [review]
Smolish typo but LGTM!

File: README.md (hunk 1 above)
Comment (0xApotheosis) [reply README.md]: Good catch! https://github.com/shapeshift/web/pull/5926/commits/8dc421f4c837ce4f8851fc49bb8e1034f660d1a4

**Pull request merged**